README.md

```diff
@@ -1,6 +1,6 @@
 # Typegoose
 
-<sub>(These badges are from typegoose:master)</sub>  
+<sub>(These badges are from typegoose:master)</sub>
```

Comment:
when possible, please revert this change

Reply:
Added a line break there to get the badges down to the next line. It seems prettier removed the trailing white space.
I'm not sure how the trailing white space got the badges to the next line earlier.

Reply:
> I'm not sure how the trailing white space got the badges to the next line earlier.

in markdown you can do an line-break in 3 way:
- add another empty line (not preferred)
- add raw html `<br/>`
- add 2 spaces at the end of the line (which was here previously)

Reply:
Oh, I didn't know the third way.